compiler/src/org.graalvm.compiler.truffle.compiler/src/org/graalvm/compiler/truffle/compiler/PartialEvaluator.java

```diff
@@ -231,6 +231,47 @@ public StructuredGraph createGraph(DebugContext debug, final CompilableTruffleAS
         return graph;
     }
 
+    @SuppressWarnings({"try", "unused"})
```

Comment:
Doesn't look it was necessary to move this method. Please put it back if possible.

Reply:
I thought about moving it to the other public methods, as I had to change the modified in order to access the method for benchmarking purposes. But I will move it back if required.